core/shared/src/main/scala/zio/Ref.scala

```diff
@@ -69,22 +69,22 @@ abstract class Ref[A] extends Serializable {
    * Atomically writes the specified value to the `Ref`, returning the value
    * immediately before modification.
    */
-  final def getAndSet(a: A)(implicit trace: Trace): UIO[A] =
```

Comment:
I imagine it doesn't affect binary compatibility?

Reply:
No it's binary compatible to make a final method non-final (but not the other way around)